### PR TITLE
Use new endpoints

### PR DESCRIPTION
**Pull request by dburns-apolitical**

### CODE REVIEW

**Comment by BigBoiBurgess** [comment]
Task linked: [CU-86944ju5h Update Moropo GHA to use new API](https://app.clickup.com/t/86944ju5h)

**Comment by Foosballfan** [comment]
theres also a replacement for the netlify api endpoint https://api.moropo.com/docs#/Test%20Runs/TestRunsController_getTestRuns

File: src/index.ts [hunk 1]
```diff
@@ -186,7 +186,7 @@ const run = async (): Promise<void> => {
       });
     }
 
-    isSync && new StatusPoller(moropoUrl, testRunId, apiKey).startPolling();
+    isSync && new StatusPoller(moropoApiUrl, testRunId, apiKey).startPolling();
```
Comment (Foosballfan) [inline src/index.ts:188]: we should remove the reference to moropoUrl since it's no longer used anywhere

File: src/methods/statusPoller.ts [hunk 2]
```diff
@@ -22,15 +22,12 @@ export default class StatusPoller {
...
         {
-          method: 'POST',
-          body: JSON.stringify({
-            testRunId: this.testRunId,
-          }),
+          method: 'GET',
           headers: {
             'Content-Type': 'application/json',
-            'x-moropo-api-key': this.apiKey,
+            'x-app-api-key': this.apiKey,
```
Comment (Foosballfan) [inline src/methods/statusPoller.ts:30]: I'm not sure if the headers are case sensitive but makes sense to align the capitilisation

**Review by Foosballfan — APPROVED** [review]

**Pull request merged**